service/src/main/java/io/camunda/service/entities/FlownodeInstanceEntity.java

```diff
@@ -7,4 +7,20 @@
  */
 package io.camunda.service.entities;
 
-public record FlownodeInstanceEntity() {}
```

Comment:
The term “FlowNode” is a compound word, and using proper camel case would improve readability.
I suggest using`FlowNodeInstanceEntity` which is consistent with existing naming in Operate
Same in the other classes